### DISCUSSION TITLE
Why the custom directory cloned not works

### DISCUSSION THREAD

**Discussion by lidaling** [NvChad/NvChad#2878] [General]
step 1: git clone https://github.com/NvChad/starter ~/.config/nvim && nvim
step 2: git clone git@github.com:dreamsofcode-io/neovim-python.git ~/.config/nvim/lua/custom
step 3: :MasonInstall python-lsp-server

Is anything else should to do？

**Comment by siduck** [comment]
with v2.5 there's no need of custom config, you can use the neovim config normally  just like any other neovim config

https://nvchad.com/news/v2.5_release/
